pkgs/development/compilers/gcc/patches/default.nix

```diff
@@ -143,6 +143,18 @@ in
   })
 ]
 
+## mlibc
+++ optionals (stdenv.targetPlatform.isMlibc && atLeast14) [
+  (fetchpatch {
+    url = "https://github.com/managarm/gcc/commit/092cb31af8ed12dbecc02327444f030a7c40c344.diff";
+    hash = "sha256-Ix95v0UxC73tfW2UliWiRgS0wC1wVUo5waM38qFlRFM=";
+  })
+  (fetchpatch {
+    url = "https://github.com/managarm/gcc/commit/b5cdfff8afc4a668e65541eec5b512d080116a43.diff";
+    hash = "sha256-0xbw/vRbUvsIIhXM8iyAxm2FWXDnYXblAJ1gFEgeOPg=";
+  })
+]
```

Comment:
How's the upstream situation looking for these?

Reply:
They will be upstreamed, previously they were blocked on upstreaming libtool patches IIRC but that got merged. Last time I asked they didn't have a timeline and couldn't meet the gcc 15 merge window. I'll ask again

Reply:
Okay, makes sense.